front-end/src/utils/auth.ts

```diff
@@ -0,0 +1,37 @@
+import axios from 'axios'
+import toast from '../components/UI/Toast'
+
+const login = async (usernameRef: any, passwordRef: any) => {
```

Comment:
This should have the proper type instead of just `any`

Reply:
Ok noted